2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,54 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+let cake = {
+  title: "Cake",
+  servings: 2,
+  ingredients: ["cheesecake", "milk"],
+};
+
+console.log(cake.title);
+console.log("Serves:" + " " + cake.servings);
+console.log("Ingredients:" + " " + cake.ingredients.join(", "));
+
+let yogurt = {
```

Comment:
If you're not going to change your object, use `const`. It's a good habit to have and leads to safer code